app/src/main/java/org/oppia/android/app/customview/LessonThumbnailImageView.kt

```diff
@@ -105,7 +105,8 @@ class LessonThumbnailImageView @JvmOverloads constructor(
       )
     }
     imageView.setBackgroundColor(
-      (0xff000000L or lessonThumbnail.backgroundColorRgb.toLong()).toInt()
+      resources.getColor(R.color.topic_lessons_story_summary_card_background_color)
+        or lessonThumbnail.backgroundColorRgb.toLong().toInt()
```

Comment:
Can we do this in the xml instead at every places this custom view is getting used?

Reply:
I suggest not to make changes to this file in this PR. You can file a separate issue to fix that. Keep it as it was before as this should not be part of this PR